src/ScrollSync.js

```diff
@@ -68,8 +68,13 @@ export default class ScrollSync extends Component {
   }
 
   syncScrollPositions = (scrolledPane) => {
-    const { scrollTop, scrollHeight, clientHeight,
-      scrollLeft, scrollWidth, clientWidth } = scrolledPane
+    const {
+      scrollTop, scrollHeight, clientHeight,
```

Comment:
Thought its easier to read this way as the properties are grouped by context (`...width` / `...height`) but okay 😉